sklearn/multiclass.py

```diff
@@ -183,11 +183,25 @@ class OneVsRestClassifier(MultiOutputMixin, ClassifierMixin,
         Coefficient of the features in the decision function. This attribute
         exists only if the ``estimators_`` defines ``coef_``.
 
+        .. deprecated:: 0.24
+            This attribute is deprecated in 0.24 and will
+            be removed in 0.26. If you use this attribute
+            in :class:`~sklearn.feature_selection.RFE` or
+            :class:`~sklearn.feature_selection.SelectFromModel`,
+            change by the importance_getter parameter instead.
```

Comment:
Is this a little clearer?

```suggestion
            be removed in 0.26. If you use this attribute
            in :class:`~sklearn.feature_selection.RFE` or
            :class:`~sklearn.feature_selection.SelectFromModel`,
            you may pass a callable to the `importance_getter`,
            that extracts the feature importances from `estimators_`.
```

Reply:
I think so.

Thank you @thomasjpfan for your suggestion!